+++
title = "Role of Phylogenetic Structure in the Dynamics of Coastal Viral Assemblages"
date = 2021-05-11T00:00:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["**Gustavsen, JA** and Suttle, CA"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "Applied and Environmental Microbiology, Vol. 87, No. 11"
publication_short = ""

# Abstract.
abstract = "Marine microbes, including viruses, are an essential part of the marine ecosystem, forming the base of the food web and driving biogeochemical cycles. Within this system, the composition of viral assemblages changes markedly with time, and some of these changes are repeatable through time; however, the extent to which these dynamics are reflected within versus among evolutionarily related groups of viruses is largely unexplored. To examine these dynamics, changes in the composition of two groups of ecologically important viruses and communities of their potential hosts were sampled every 2 weeks for 13-months at a coastal site in British Columbia, Canada. We sequenced two marker genes for viruses—the gene encoding the major capsid protein of T4-like phages and their relatives (gp23) and the RNA-dependent RNA polymerase (RdRp) gene of marnavirus-like RNA viruses—as well as marker genes for their bacterial and eukaryotic host communities, the genes encoding 16S rRNA and 18S rRNA. There were strong lagged correlations between viral diversity and community similarity of putative hosts, implying that the viruses influenced the composition of the host communities. The results showed that for both viral assemblages, the dominant clusters of phylogenetically related viruses shifted over time, and this was correlated with environmental changes. Viral clusters contained many ephemeral taxa and few persistent taxa, but within a viral assemblage, the ephemeral and persistent taxa were closely related, implying ecological dynamics within these clusters. Furthermore, these dynamics occurred in both the RNA and DNA viral assemblages surveyed, implying that this structure is common in natural viral assemblages."

# Summary. An optional shortened abstract.
summary = ""

# Digital Object Identifier (DOI)
doi = "doi.org/10.1128/AEM.02704-20"

# Is this a featured publication? (true/false)
featured = false

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["virus","phylogeny","coastal","time series","bacteria","phytoplankton","seed bank","18S rRNA","16S rRNA","dynamics","Picornavirales","Myoviridae"]

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Links (optional).
url_pdf = "https://journals.asm.org/doi/full/10.1128/aem.02704-20"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# links = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
